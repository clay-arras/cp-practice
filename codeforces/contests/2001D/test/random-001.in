1
8
12 4 14 3 25 9 4 11
