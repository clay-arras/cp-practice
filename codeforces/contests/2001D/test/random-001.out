7
12 4 14 3 25 9 11
